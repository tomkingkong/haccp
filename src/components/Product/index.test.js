import React from 'react';
import { shallow } from 'enzyme';

import { Product } from '.';

describe('Product Component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Product />);
    expect(wrapper).toMatchSnapshot();
  });
});