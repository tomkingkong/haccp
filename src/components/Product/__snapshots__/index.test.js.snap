// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Component should match snapshot 1`] = `
<article
  className="user-product"
>
  <h3 />
  <WithStyles(List)
    component="nav"
  >
    <WithStyles(Divider) />
    <WithStyles(ListItem)
      aria-label="Edit Ingredients Information"
      button={true}
      divider={true}
      id="Ingredients"
      key="0"
      onClick={[Function]}
      style={
        Object {
          "fontSize": "9rem",
          "paddingLeft": "0.5rem",
          "paddingRight": 0,
        }
      }
    >
      <WithStyles(ListItemText)
        primary="Ingredients"
        style={
          Object {
            "zIndex": "-1",
          }
        }
      />
      <pure(AddIcon)
        style={
          Object {
            "color": "green",
            "fontSize": 20,
            "zIndex": "-1",
          }
        }
      />
    </WithStyles(ListItem)>
    <WithStyles(ListItem)
      aria-label="Edit Receiving Information"
      button={true}
      divider={true}
      id="Receiving"
      key="1"
      onClick={[Function]}
      style={
        Object {
          "fontSize": "9rem",
          "paddingLeft": "0.5rem",
          "paddingRight": 0,
        }
      }
    >
      <WithStyles(ListItemText)
        primary="Receiving"
        style={
          Object {
            "zIndex": "-1",
          }
        }
      />
      <pure(EditIcon)
        style={
          Object {
            "color": "grey",
            "fontSize": 15,
            "zIndex": "-1",
          }
        }
      />
    </WithStyles(ListItem)>
    <WithStyles(ListItem)
      aria-label="Edit Inventory Information"
      button={true}
      divider={true}
      id="Inventory"
      key="2"
      onClick={[Function]}
      style={
        Object {
          "fontSize": "9rem",
          "paddingLeft": "0.5rem",
          "paddingRight": 0,
        }
      }
    >
      <WithStyles(ListItemText)
        primary="Inventory"
        style={
          Object {
            "zIndex": "-1",
          }
        }
      />
      <pure(EditIcon)
        style={
          Object {
            "color": "grey",
            "fontSize": 15,
            "zIndex": "-1",
          }
        }
      />
    </WithStyles(ListItem)>
    <WithStyles(ListItem)
      aria-label="Edit Processing Information"
      button={true}
      divider={true}
      id="Processing"
      key="3"
      onClick={[Function]}
      style={
        Object {
          "fontSize": "9rem",
          "paddingLeft": "0.5rem",
          "paddingRight": 0,
        }
      }
    >
      <WithStyles(ListItemText)
        primary="Processing"
        style={
          Object {
            "zIndex": "-1",
          }
        }
      />
      <pure(EditIcon)
        style={
          Object {
            "color": "grey",
            "fontSize": 15,
            "zIndex": "-1",
          }
        }
      />
    </WithStyles(ListItem)>
    <WithStyles(ListItem)
      aria-label="Edit Packaging Information"
      button={true}
      divider={true}
      id="Packaging"
      key="4"
      onClick={[Function]}
      style={
        Object {
          "fontSize": "9rem",
          "paddingLeft": "0.5rem",
          "paddingRight": 0,
        }
      }
    >
      <WithStyles(ListItemText)
        primary="Packaging"
        style={
          Object {
            "zIndex": "-1",
          }
        }
      />
      <pure(EditIcon)
        style={
          Object {
            "color": "grey",
            "fontSize": 15,
            "zIndex": "-1",
          }
        }
      />
    </WithStyles(ListItem)>
    <WithStyles(ListItem)
      aria-label="Display Summary"
      button={true}
      divider={true}
      id="Summary"
      key="5"
      onClick={[Function]}
      style={
        Object {
          "fontSize": "9rem",
          "paddingLeft": "0.5rem",
          "paddingRight": 0,
        }
      }
    >
      <WithStyles(ListItemText)
        primary="Summary"
        style={
          Object {
            "zIndex": "-1",
          }
        }
      />
      <pure(PrintIcon)
        style={
          Object {
            "color": "#df6e21",
            "zIndex": "-1",
          }
        }
      />
    </WithStyles(ListItem)>
  </WithStyles(List)>
</article>
`;
